### ISSUE TITLE
pretraining logs

### ISSUE THREAD

**Issue by joez17** [klauscc/VindLU#2]
Hi! I'm trying to pretrain VindLU using 5M data, can you provide the pretraining logs for reference? Thanks!

**Comment by joez17** [comment]
And when i use the given 5M checkpoint to finetune on msrvtt-9k retrieval task, the result is lower than given results. Fro example, R@1 is 39.7 for best.

**Comment by klauscc** [comment]
Did you run the evaluation code after fine-tuning? The validation results during finetuning are evaluated with 4 frames for faster training. We evaluate our final results using 12 frames.

**Comment by joez17** [comment]
> Did you run the evaluation code after fine-tuning? The validation results during finetuning are evaluated with 4 frames for faster training. We evaluate our final results using 12 frames.

That's right! Thanks! And can you provide pretraining logs for further reference?
